Makefile

```diff
@@ -11,8 +11,9 @@ typecheck:
 coverage:
 	python3 -m pytest --cov nio --benchmark-disable
 
-isort:
-	isort -p nio
+lint:
+	black nio
+	isort --profile=black nio
```

Comment:
Fair enough! Honestly, I like the pre-commit hook, and have already started using it! Saves a whole command so that's cool with me! I removed the `lint` target out of the Makefile